### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by janpanschab**
Great first step to improve documentation from @kettanaito.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
 ## Features
+
+  | Pre-processing | Quality control | Post-processing
+------------ | ------------- | ------------- | -------------
+**CSS** | [SASS](https://github.com/dlmanning/gulp-sass) | Stylelint | **[PostCSS](https://github.com/postcss/postcss)** ([autoprefixer](https://github.com/postcss/autoprefixer),  [cssnano](https://github.com/ben-eb/cssnano)) <br>[Sourcemaps](https://github.com/floridoo/gulp-sourcemaps)
+**JavaScript** | Babel | [ESLint](http://eslint.org) | [Browserify](http://browserify.org) <br>[Watchify](https://github.com/substack/watchify) <br>[Uglify](https://github.com/terinjokes/gulp-uglify)
+**SVG** | | | svgmin <br>[SVGStore](https://github.com/w0rm/gulp-svgstore)
+**HTML** | [Nunjucks](https://github.com/sindresorhus/gulp-nunjucks) | | [Prettify](https://github.com/jonschlinkert/gulp-prettify)
+
 - **CSS:**
```
Comment (vbulant) [inline README.md]: CSS, JS, HTML and Icons should be removed as they were moved to the table above.

File: README.md [hunk 2]
```diff
@@ -24,28 +40,89 @@ Front-end gulp dev stack.
...
 
-## Usage
+## Getting started
+### Preparations
+1. This development stack uses [Node](https://nodejs.org), so make sure you have it installed.
+2. Install [Gulp](http://gulpjs.com) globally:  
+```bash
+npm install -g gulp
+```
+**NOTE:** If you have already installed Node and Gulp on your machine, you may skip this step.
```
Comment (vbulant) [inline README.md]: I think everyone knows this so it can be removed (**NOTE**).

File: README.md [hunk 3]
```diff
@@ -24,28 +40,89 @@ Front-end gulp dev stack.
...
+
+## Troubleshooting
+Common errors  |
+------------ |
+**Error:**  "*Local gulp not found in ...*" when running `gulp`. |
+**Solution:** Make sure you run `npm install` after cloning the repository. |
 
-Run `gulp` and pick built assets from `dist` folder.
\ No newline at end of file
+When you have encountered a bug, or have a useful suggestion how to improve this development stack, do not hesitate to [Create a new issue]().
```
Comment (vbulant) [inline README.md]: Link on "create a new issue" doesn’t work.

**Review by vbulant — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (janpanschab) [reply README.md]: OK. I will only keep icons there because of the description.

**Review by vbulant — APPROVED** [review]

**Pull request merged**